src/components/profile-details-list-item/ProfileDetailsListItem.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { Preview } from "../preview";
+
+import { SProfileDetailsListItem } from "./styles/SProfileDetailsListItem";
+import { SProfileDetailsListItemTitle } from "./styles/SProfileDetailsListItemTitle";
+import { SProfileDetailsListItemLabel } from "./styles/SProfileDetailsListItemLabel";
+import { SProfileDetailsListItemPreviewWrapper } from "./styles/SProfileDetailsListItemPreviewWrapper";
+import { SProfileDetailsListItemPreviewStatus } from "./styles/SProfileDetailsListItemPreviewStatus";
+
+import { IPreview } from "../preview/Preview";
+
+export enum IProfileDetailsListItemStatus {
+  online = "online",
+  offline = "offline",
+}
+
+interface IProfileDetailsListItem {
+  title?: string;
+  label?: string;
+  previewProps?: IPreview;
+  status?: IProfileDetailsListItemStatus;
+}
+
+export const ProfileDetailsListItem: React.FC<IProfileDetailsListItem> = ({ title, label, previewProps, status }) => {
+  return (
+    <SProfileDetailsListItem>
```

Comment:
@bredek this seems to be clickable, there is missing a onClick event and also a pointer cursor.